index.html

```diff
@@ -90,25 +69,89 @@
 
     <script>
 
-      document.querySelector('.ramda-repl-target').addEventListener('click', function(e) {
+      (function() {
 
-        ramdaRepl(e.target, {
 
-          apiUrl: 'https://www.googleapis.com/urlshortener/v1/url?key=AIzaSyDhbAvT5JqkxFPkoeezJp19-S_mAJudxyk',
-          returnUrl: 'http://ramdajs.com/repl/',
+        var ramdaUrl;
 
-          // If unset, initialValue will use the value or textcontent of the
-          // target element.
-          initialValue: URI.decode(window.location.hash).substring(2),
-
-          // Called with the pre-compiled content
-          onChange : function(code) {
-            window.location.hash = '?' + URI.encode(code);
+        var rx = /^\d+\.\d+(\.\d+)?$/;
+        var version = 'latest';
+        location.search.substr(1).split('&').forEach(function(pair) {
```

Comment:
Ok! [Here is an attempt using reduce] (https://github.com/ramda/repl/pull/25/commits/32f7316148f3b0e9f4a1360d19c5a0923eec3c75). :octopus: